rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -295,6 +290,27 @@ namespace rocal{
         return py::cast<py::none>(Py_None);
     }
 
+    '''
```

Comment:
Have you checked by building this file?
It is throwing errors for me as you have used python style of commenting in this Cpp file please check and change this